pages/deployments/browser-quickstart.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Plural CD In-Browser Quickstart
+description: Deploying your services using the Plural Console UI.
+---
+
+## Overview
+
+This guide goes over how to deploy your services with Plural using the Plural Console UI. At the end of this tutorial, you will have:
+
+- Provisioned new clusters, and/or deployed the Plural Deployment Operator on your existing clusters.
+- Imported the Git repositories containing your code and manifests.
+- Deployed your code onto your clusters of choice.
+- Optionally updated any configurations and permissions for the clusters and services.
+
+in under 30 minutes, all within your browser. You can see the process in the video here or follow the instructions step-by-step:
+
+{% embed url="https://www.youtube.com/watch?v=jBlq45ntbBw" aspectRatio="16 / 9" /%}
+
+## Onboard to Plural and install the Plural Console
+
+If you haven't already, you'll need to follow the Plural guide to install Console. You can use the guide for the [in-browser Cloud Shell](/getting-started/cloud-shell-quickstart) or the [CLI](/getting-started/quickstart) to get started.
+
+## Log in to Console and Add Clusters
+
+Once you've logged in to Plural Console, you can click the "Continuous deployment" tab to set up your clusters. By default, you should see the cluster that your Console is deployed on.
```

Comment:
@michaeljguarino just checking that this is true, they'll always at least have their current console cluster visible here

Reply:
yep